### PR TITLE
404 and 500 error 

### PR DESCRIPTION
**Pull request by ahmedjasarevic**
Handle for not found pages.

### CODE REVIEW

File: aMovieDatabase/aMovieDatabase/settings.py [hunk 1]
```diff
@@ -26,9 +26,9 @@
 SECRET_KEY = config("SECRET_KEY")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
```
Comment (mehmedkadric) [inline aMovieDatabase/aMovieDatabase/settings.py:29]: Why did you reset DEBUG to False?

Comment (ahmedjasarevic) [reply aMovieDatabase/aMovieDatabase/settings.py:29]: If I turn DEBUG to false,  then all static files missing.
With debug turned off Django won't handle static files for you any more - your production web server should take care of that.

Comment (mehmedkadric) [reply aMovieDatabase/aMovieDatabase/settings.py:29]: Are we going to production?

Comment (ahmedjasarevic) [reply aMovieDatabase/aMovieDatabase/settings.py:29]: I don't know, we have to turn it on until we deploy and upload static files

**Pull request merged**